deployments/default/common.yml

```diff
@@ -617,3 +617,8 @@ dns_server_list: [ ]
 #
 # vsd_install_inplace_upgrade: False
 
+# < Keep root access >
+# Create custom username and password without hardening
+#
+# keep_root_access: False
```

Comment:
`keep_vsd_root_access` is a better name. The the description mention it will only be used if custom username is provided and its only for the VSD. default it to False for backward compatibility